framework/audio/internal/midisource.h

```diff
@@ -61,6 +62,7 @@ class MidiSource : public IAudioSource
     std::string m_name;
     std::shared_ptr<SL> m_sl;
     std::shared_ptr<midi::ISequencer> m_seq;
+    midi::ISequencer::Context* m_seqContext = nullptr;
```

Comment:
why not unique_ptr?

Reply:
Changed on shared_ptr (it is shared between class)